README.md

```diff
@@ -146,7 +146,11 @@ For "static" feature flags we set the flag values in env vars.
 To add a new feature flag, set a `"#{flag_name.upcase}_FEATURE_FLAG"` env var with value `"ENABLED"` in all environments where you want the flag enabled.
 Then add `flag_name` to the list of flags  in `app/lib/feature_flags.rb`.
 
-To use the feature flag in your code, just call `FeatureFlags.enabled?(:flag_name)`.
+When adding a `flag_name` to the list of static flags, you will need to decide if this is a `"global"` flag i.e. always taken from the env var and not the session, or a `"session"` flag i.e. taken from the `session_data` of the check. 
+
+We introduced this as a way of making our feature flags 'backwards compatible' - if a user check is underway during the switch-on of a flag, their user journey will not be affected by any flag-related changes. This is because we use their `session_data` to determine the value of the flag that was set at the start of their check. 
+
+To use the feature flag in your code, you will need to call `def enabled?`. In order to check if a flag is enabled on a global level, you will need to call `FeatureFlags.enabled?(:flag_name, without_session_data: true)`. In order to use the flag value set in the session, call `FeatureFlags.enabled?(:flag_name, session_data)`. `session_data` is an optional argument. If you do not wish to pass in `session_data` to determine the flag value, you must confirm this by passing in `without_session_data: true`.
```

Comment:
The bit about passing in `without_session_data: true` for global checks is confusing to me. From the discussion, I would expect that only to be used when you _couldn't_ pass session data, which to me is a bit different.

Reply:
I think I have written this README in a confusing way. So yes I guess the only time you wouldn't pass session_data is when you don't have it, e.g. on the start pages. So in those cases you'd have `FeatureFlags.enabled?(:flag, without_session_data: true)` - if you just pass `FeatureFlags.enabled?(:flag)` you'll get an error.

On reflection I have written something really confusing and we should just say 'always pass `session_data` if you can'.

Reply:
I guess I was thinking there might be cases where you know you only care about a global flag value, which won't be in the `session_data` so there isn't any point to passing that in. But that probably just makes things less clear!

Reply:
I'll reword this 😅

Reply:
@exonian have updated the README, let me know if it now sounds clearer!